### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by Demihov**

### CODE REVIEW

File: App.Cards/Controllers/CardsController.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            _cardsManager = cardsManager;
+        }
+
+        
+        [HttpGet("get")]
+        public ActionResult<Card> Get(long number, DateTime expiresEnd, ushort CVV)
+        {
+            var serviceCallResult = _cardsManager.GetCard(number, expiresEnd, CVV);
+
+            if (serviceCallResult == null)
```
Comment (Demihov) [inline App.Cards/Controllers/CardsController.cs]:
```suggestion
            if (serviceCallResult)
```

File: App.Cards/Controllers/CardsController.cs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                return NotFound();
+            return serviceCallResult;
+        }
+
+        [HttpPut("block")]
+        public ActionResult<bool> BlockCard(string ownerName, long number, DateTime expiresEnd, ushort CVV)
+        {
+            var serviceCallResult = _cardsManager.BlockCard(ownerName, number, expiresEnd, CVV);
+
+            if (serviceCallResult == true)
```
Comment (Demihov) [inline App.Cards/Controllers/CardsController.cs]:
```suggestion
            if (serviceCallResult)
```

File: App.Cards/Repositories/InMemoryCardsRepository.cs [hunk 3]
```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using App.Configuration;
+using App.Models.Cards.Models;
+using App.Repositories;
+
+namespace App.Cards.Repositories
+{
+    public class InMemoryCardsRepository: ICardsRepository, ITransientDependency
```
Comment (Demihov) [inline App.Cards/Repositories/InMemoryCardsRepository.cs]: think how not repeat yourself and reuse logic

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
Please, resolve conflicts

**Comment by ghost** [comment]
still not resolved :)

**Pull request merged**